static/manifests/calico/ConfigMap/calico-config.yaml

```diff
@@ -32,10 +32,8 @@ data:
           "nodename": "__KUBERNETES_NODE_NAME__",
           "mtu": __CNI_MTU__,
           "ipam": {
-              {{ if .DualStack }}
-              "assign_ipv4": "true",
-              "assign_ipv6": "true",
-              {{ end }}
+              "assign_ipv4": "{{ .EnableIPv4 }}",
```

Comment:
This wasn't set before for IPv4 only clusters. Is this that okay?

Reply:
It is okay, but redundant. The default values are true for ipv4 and false for ipv6. We could do instead:

```
              {{ if ! .EnableIPv4 }}
              "assign_ipv4": "false",
              {{ end }}
              {{ if 6 .EnableIPv6 }}
              "assign_ipv6": "true",
              {{ end }}
```

I think the template looks better this way. I'm gonna merge it as is because you're out of office today and I need this merged to keep working on CPLB, but if you want we can change it in a subsequent PR